### PR TITLE
librealsense: Fix package QA warnings: 'No GNU_HASH in the elf binary:'

### PR DESCRIPTION
**Pull request by avalluri**
Signed-off-by: Amarnath Valluri amarnath.valluri@intel.com

### CODE REVIEW

File: recipes-support/librealsense/librealsense_1.10.0.bb [hunk 1]
```diff
@@ -88,3 +88,10 @@ FILES_${PN}-tests += "\
 
 # Configuration is not required
 do_configure[noexec] = "1"
+
+#Skip ldflags QA check
```
Comment (mythi) [inline recipes-support/librealsense/librealsense_1.10.0.bb]: The problem is that the build system LDFLAGS are not passed to make. 

Proper fix is to ```LDFLAGS=${LDFLAGS}```  above.

Comment (avalluri) [inline recipes-support/librealsense/librealsense_1.10.0.bb]: @mythi I tried by passing LDFLAGS to make as below(in bbappend):
    EXTRA_OEMAKE_append = " 'LDFLAGS=${LDFLAGS}'"
Still that couldn't solve the issue :(

Comment (mythi) [inline recipes-support/librealsense/librealsense_1.10.0.bb]: Yeah. It seems librealsense/Makefile does not use LDFLAGS at all. There's a PR pending to move to CMake but that does not take LDFLAGS either.

Looks like this is the only way until librealsense itself is fixed.

**Comment by avalluri** [comment]
@mythi, Now i patched the source to respect LDFLAGS. I think this is better solution than my initial patch of skipping QA. ?

**Comment by mythi** [comment]
@avalluri Thanks! Yes this is much better.

**Comment by avishaymahluf** [comment]
@avalluri, This patch is now part of librealsense v1.11.2 and since we're going to update the librealsense recipe to v.1.11.2 this patch is no longer needed.

Thanks

**Comment by avalluri** [comment]
@avishaymahluf Yes i agree, the issue fixed in v1.11.2, but for some reason the fix(https://github.com/IntelRealSense/librealsense/commit/75ffaf57c77798832ce7d6afb8031944b12bc8f8) is not present on librealsense master :!. Anyways i agree with you, this patch is no longer needed. Thanks

**Pull request closed without merging**